src/services/resources/logic.ts

```diff
@@ -113,4 +130,21 @@ export const neighbourhoodVolunteersLogic = async (neighbourhood: string): Promi
         statusCode: 200,
     };
     return response;
+};
+
+export const updateVolunteerLogic = async (userId: string, fields: IUpdateRequest): Promise<ILogicResponse> => {
```

Comment:
This is dangerous - we need to tighten this up before we release it.

In the current logic, I think:
1. Any user can edit any contacts details - we need to stick this behind the auth endpoint
2. Assuming this is behind captain auth, any captain can edit any contact (including other captain's and other captain's volunteers)
3.  Assuming this had auth, any user could edit any contact field, including if they are a vol, their neighbourhood, even our primary key (email) - we need to lock this down to only being able to update notes